src/odemis/dataio/tiff.py

```diff
@@ -1271,6 +1309,13 @@ def _addImageElement(root, das, ifd, rois, fname=None, fuuid=None):
         except Exception:
             logging.warning("Spectrum metadata is insufficient to be saved")
 
+    time_list = None
+    if set(globalMD.keys()) & {model.MD_TIME_LIST}:
```

Comment:
For checking a single metadata, you can just write `if MD_TIME_LIST in globalMD:`